traceapp/tmpl/data.go

```diff
@@ -6,16 +6,15 @@ import (
 	"go/build"
 	"log"
 	"net/http"
-	"path/filepath"
 )
 
-func defaultBase(path string) string {
-	p, err := build.Import(path, "", build.FindOnly)
+func importPathToDir(importPath string) string {
+	p, err := build.Import(importPath, "", build.FindOnly)
 	if err != nil {
-		log.Fatal(err)
+		log.Fatalln(err)
 	}
 	return p.Dir
 }
 
 // Data is a virtual filesystem that contains template data used by Appdash.
-var Data = http.Dir(filepath.Join(defaultBase("sourcegraph.com/sourcegraph/appdash/traceapp/tmpl"), "data"))
+var Data = http.Dir(importPathToDir("sourcegraph.com/sourcegraph/appdash/traceapp/tmpl/data"))
```

Comment:
Yep. Although it's not related to the vfsgen binary change, this is just updating this code to the latest version of this pattern. I should've made it a separate PR, that way it could've been merged much sooner (you're welcome to cherry pick it yourself).